sound_play/scripts/soundplay_node.py

```diff
@@ -317,7 +317,7 @@ def callback(self,data):
     # Purge sounds that haven't been played in a while.
     def cleanupdict(self, dict):
         purgelist = []
-        for (key,sound) in dict.iteritems():
+        for (key,sound) in dict.items():
```

Comment:
can you change to `iter(dict.items())` instead to follow PEP469?
https://www.python.org/dev/peps/pep-0469/

Reply:
I followed the ROS guidelines https://wiki.ros.org/python_2_and_3_compatible_code#dict.iter.2A.28.29 but am fine either way

Reply:
```suggestion
        for (key,sound) in iter(dict.items()):
```